### PR TITLE
[ui][new] adding test for Slack consumer

### PR DESCRIPTION
**Pull request by mkralik3**
- The test for the Slack consumer was added
- The CSS selectors in the BasicFilter were updated
- SlackConnector was updated according to updated [credentials](https://gitlab.cee.redhat.com/jboss-fuse-qe/ci-ops-fuse/commit/2fea1870138ebdbde9da859f3ac36fb1c851ac94?merge_request_iid=146). 

//test: @slack-to-db

### CODE REVIEW

**Comment by syndesisqe** [comment]
Can one of the admins verify this patch?

File: ui-tests/src/test/resources/features/integrations/slack.feature [hunk 1]
```diff
@@ -135,3 +136,62 @@ Feature: Slack Connector
...
+    When navigate to the "Integrations" page
+    Then Integration "slack-to-db" is present in integrations list
+    And wait until integration "slack-to-db" gets into "Running" state
+
+    When send message "Red Hat testSlack" on channel "test"
+    And send message "Red Hat test incorrect Slack" on channel "test"
+    And sleep for "10000" ms
+
+    Then checks that query "select * from contact where company = 'Red Hat testSlack' AND first_name = 'syndesis-bot'" has some output
+    Then checks that query "select * from contact where company = 'Red Hat test incorrect Slack'" has not some output
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/slack.feature]: change the wording to "... has no output"

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java [hunk 2]
```diff
@@ -188,6 +188,11 @@ public void checkValuesExistInTable(String query) {
         assertThat(dbUtils.getCountOfInvokedQuery(query)).isGreaterThanOrEqualTo(1);
     }
 
+    @And("^.*checks? that query \"([^\"]*)\" has not some output$")
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java]: change "And" to "Then"

File: utilities/src/main/java/io/syndesis/qe/utils/SlackConnector.java [hunk 3]
```diff
@@ -35,10 +35,10 @@
      * is API Token for slack workspace custom integration called "Bots".
      */
     public SlackConnector() {
-        Optional<Account> optional = AccountsDirectory.getInstance().getAccount("QE Slack");
+        Optional<Account> optional = AccountsDirectory.getInstance().getAccount("QE Slack consumer");
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/utils/SlackConnector.java]: rename "optional" to "account" or something like that :-)

**Comment by avano** [comment]
ok to test

**Comment by mkralik3** [comment]
Fixed

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
@mkralik3 please rebase your branch on current master - I pushed the syndesis version change to 1.5 (when there is 1.4, the tests run against latest 7.1 version that does not have the slack consumer afaik)

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
Done

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
@mkralik3 by the way, I changed your first comment to include `//test: @something` to run the `@something` tag in the job.. just for the future :-)

**Pull request merged**